tests/test_fill.py

```diff
@@ -0,0 +1,61 @@
+import dpctl
+import numpy as np
```

Comment:
Could you please extend `TEST_SCOPE` in `.github/workflows/conda-package.yml` with `test_fill.py`